### PR TITLE
Pr 10 cleaning

### PR DESCRIPTION
**Pull request by Voland91**

### CODE REVIEW

File: src/components/Event/Event.tsx [hunk 1]
```diff
@@ -1,72 +1,37 @@
...
-      <DescriptionWrapper>
-        <Title title={title} />
-        <div>{description}</div>
-      </DescriptionWrapper>
-      <Button/>
-    </EventWrapper>
-  );
+const Event: FC<EventProps> = ({ title, description, background, eventDate }) => {
+    const year = eventDate.getFullYear();
+    const month = eventDate.getMonth() + 1;
```
Comment (RafalSzczuka) [inline src/components/Event/Event.tsx]: seems weird ? month value + 1 ?

File: src/components/Event/Event.tsx [hunk 2]
```diff
@@ -1,72 +1,37 @@
...
+    const day = eventDate.getDate();
+    console.log(eventDate);
```
Comment (RafalSzczuka) [inline src/components/Event/Event.tsx]: console looooooooooog

File: src/components/Event/Event.tsx [hunk 3]
```diff
@@ -1,72 +1,37 @@
...
+
+    return (
+        <EventWrapper background={background}>
+            <DateWrapper>
+                <div>
+                    <ColoredYear>{year}</ColoredYear>
+                    {` / ${month < 10 ? `0${month}` : month} / ${day < 10 ? `0${day}` : day}`}
```
Comment (RafalSzczuka) [inline src/components/Event/Event.tsx]: umm, quite unusual conditions. Consider make local string from date object and parse it accordingly, e.g:

```suggestion
  const year = date.getFullYear();
  const month = date.toLocaleDateString().split('/')[1];
  const day = date.toLocaleDateString().split('/')[0];
  const time = date.toLocaleTimeString();
```

File: src/components/Event/Event.tsx [hunk 4]
```diff
@@ -1,72 +1,37 @@
...
+                </div>
+                <div>20:41:06 GMT</div>
```
Comment (RafalSzczuka) [inline src/components/Event/Event.tsx]: why time is hardcoded? in previous version it was parsed from date object

**Review by RafalSzczuka — CHANGES_REQUESTED** [review]

File: src/components/Event/Event.tsx (hunk 1 above)
Comment (Voland91) [reply src/components/Event/Event.tsx]: added comment for that

File: src/components/Event/Event.tsx (hunk 2 above)
Comment (Voland91) [reply src/components/Event/Event.tsx]: my bad, shame on me

File: src/components/Event/Event.tsx (hunk 3 above)
Comment (Voland91) [reply src/components/Event/Event.tsx]: created new const + comment

File: src/components/Event/Event.tsx (hunk 4 above)
Comment (Voland91) [reply src/components/Event/Event.tsx]: I forgot about that, repaired at the new commit

**Review by RafalSzczuka — APPROVED** [review]

**Pull request merged**